components/filter/filter.js

```diff
@@ -697,7 +697,7 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				}
 				this._activeFiltersSubscribers.updateSubscribers();
 			} else if (prop === 'values') {
-				if (dimension.searchValue) shouldSearch = true;
+				if (dimension.searchValue || dimension.searchType === 'full-manual') shouldSearch = true;
```

Comment:
Previously it was making the assumption that if search is empty, then all values will be shown but we don't want this in `full-manual`. This doesn't trigger a search event but will re-filter the items based on the results of the previous search.